### ISSUE TITLE
training code

### ISSUE THREAD

**Issue by zhongchenzhao** [HiLab-git/SCPM-Net#5]
Hi, I'm very interested in your work. Could you possibly make your entire training code public?

**Comment by Luoxd1996** [comment]
Hi, 
Due to privacy concerns, this work is supported by a company, now we just can release the code of the proposed network and loss function. But we are working to release the entire project, it requires some time, you know too many documents need to be approved.
Best,
Xiangde.

**Comment by zhongchenzhao** [comment]
Thank you for the quick reply!
